cmd/shipperctl/cmd/clean.go

```diff
@@ -0,0 +1,238 @@
+package cmd
```

Comment:
nit: I would rename the filename to something like `decommission.go` since it has both "clean" and "count" commands.